frame/contracts/src/migration.rs

```diff
@@ -47,6 +48,11 @@ pub fn migrate<T: Config>() -> Weight {
 		StorageVersion::new(6).put::<Pallet<T>>();
 	}
 
+	if version < 7 {
+		weight = weight.saturating_add(v7::migrate::<T>());
+		StorageVersion::new(7).put::<Pallet<T>>();
```

Comment:
Hmm, is this the second write being accounted for in the `v7::migrate` weights?

Reply:
No.  See above. This write is actually unaccounted for.